the-collab-lab/tcl-71-smart-shopping-list
# Adjust padding on pages, buttons, input fields, width component, responsiveness

## Description

This PR continues to add consistency to the website, it adjusts:
-the padding at the top of the page and the bottom
-the width of the main component on each page
-the padding of buttons and input fields
-The responsiveness on the login page is improved

Note: There will still be some adjustement to do once this PR is merged with the other PR's currently waiting for review.

## Related Issue

closes #55 

## Acceptance Criteria

- [X] pages are responsive
- [X] padding buttons consistent throught the website
- [X] box shadow consistent throught the website
- [X] width elements consistent throught the website (ex: buttons, input fields...etc)
- [X] padding-top pages consistent throught the website

## Type of Changes

`enhancement`

## Updates

### Before


![Screenshot 2024-04-02 203511](https://github.com/the-collab-lab/tcl-71-smart-shopping-list/assets/114222588/d57921b8-d410-4f0e-93c3-92365471212d)
![Screenshot 2024-04-02 203439](https://github.com/the-collab-lab/tcl-71-smart-shopping-list/assets/114222588/bcb10eb8-347a-46ff-b54c-ac2a958f8188)
![Screenshot 2024-04-02 203258](https://github.com/the-collab-lab/tcl-71-smart-shopping-list/assets/114222588/61d4844b-dde7-4ebc-8f7c-70eda1c5e5d6)
![Screenshot 2024-04-02 201319](https://github.com/the-collab-lab/tcl-71-smart-shopping-list/assets/114222588/26b94ae8-d30b-4cb6-bc2c-88897c2cf79b)
![Screenshot 2024-04-02 201246](https://github.com/the-collab-lab/tcl-71-smart-shopping-list/assets/114222588/79de0b88-d50e-4f46-8786-8285dd60e707)

### After
![Screenshot 2024-04-02 223832](https://github.com/the-collab-lab/tcl-71-smart-shopping-list/assets/114222588/48aeb8b4-be39-483c-917f-352e62c51ffd)
![Screenshot 2024-04-02 223728](https://github.com/the-collab-lab/tcl-71-smart-shopping-list/assets/114222588/1eeb2f4a-e418-4bdf-b0d3-ae86de1cf914)
![Screenshot 2024-04-02 223713](https://github.com/the-collab-lab/tcl-71-smart-shopping-list/assets/114222588/cbc48c21-6472-4dae-b779-386d53ba900e)
![Screenshot 2024-04-02 223657](https://github.com/the-collab-lab/tcl-71-smart-shopping-list/assets/114222588/4ac77a32-8f56-4c94-aa32-83fce9a11e1c)
![Screenshot 2024-04-02 223643](https://github.com/the-collab-lab/tcl-71-smart-shopping-list/assets/114222588/ab39a6e1-ce00-47de-8635-0fd44b323592)


## Testing Steps / QA Criteria

- git checkout -b ce-issue-29
- git pull ce-issue-29
Use your devtools to controle the responsiveness, and check all pages for the consistency of the visual.
